omero/developers/json-api.rst

```diff
@@ -319,6 +319,10 @@ cookie of any request or from the following URL:
       "data": "eNoVq528bOqlhQqbCzKuviODTRX3PUO2"
     }
 
+If you are using the API over HTTPS, you will also need to set the ``Referer`` key in 
+request headers when supplying the CSRF token to other endpoints. The referer URI 
```

Comment:
This looks like very useful info which I think I would have found useful in the past when failing to fix CORS issues! 

Just reading about the miss-spelling of the ``Referer`` header key at https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy 
"The original header name [Referer](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) is a misspelling of the word "referrer". The Referrer-Policy header does not share this misspelling."

So instead of "referer" in this line, you could use "referrer" but it might be clearer to repeat the ``Referer``.